src/util/time_lua.inl

```diff
@@ -374,6 +374,28 @@ namespace {
     }
 }
 
+/**
+ * Returns the number of seconds between the provided start time and end time. If the end
+ * time is before the start time, the return value is negative. If the start time is equal
+ * to the end time, the return value is 0.
+ * Both start and end times must be valid ISO 8601 date strings.
+ */
+[[codegen::luawrap]] double duration(std::string start, std::string end) {
+    using namespace openspace;
+
+    const double tStart = Time::convertTime(start);
+    const double tEnd = Time::convertTime(end);
+    return tEnd - tStart;
```

Comment:
Do we want to add an `abs` here? In case the user mixes up start and end, or do we put it on the user to make sure it is correct together with the comment above?

Reply:
I'd rather leave it up to the user.  It'd be different if the parameters would already be in a form that would support quickly checking whether the start time is before the end time, but a user would have to call the convertTime function manually to check it in case they are interested in knowing what the ordering of the values is